doc/content/getting-started/migrating/faq.md

```diff
@@ -6,27 +6,27 @@ This section answers frequently asked questions regarding migration to {{% tts %
 
 ### Why should I migrate my devices and gateways to {{% tts %}}? 
 
-{{% tts %}} is more scalable, more secure, and supports more of the LoRaWAN specification than {{% ttnv2 %}}. The end of life for {{% ttnv2 %}} environment is planned for December 1, 2021.
+{{% tts %}} is more scalable, more secure, and supports more of the LoRaWAN specification than {{% ttnv2 %}}. The Things Network {{% ttnv2 %}} machines have been completely shut down on **December 8, 2021**. Only private The Things Industries {{% ttnv2 %}} deployments might still be running, but the associated software will no longer be developed or maintained by The Things Industries.
 
 ### When should I migrate my devices and gateways to {{% tts %}}? 
 
-Start migrating your devices and gateways as soon as possible! {{% ttnv2 %}} is now read-only so you cannot add new gateways, devices and applications anymore, and the final deadline for migrating is December 1, 2021, when {{% ttnv2 %}} will be shut down.
+There is no migration deadline for private {{% ttnv2 %}} deployments to migrate, but since there is no official support anymore, the sooner the better.
```

Comment:
I think that the `Start migrating` message should stay. We want to relay the importance of this action.